### DISCUSSION TITLE
Slow Snaking Sliders

### DISCUSSION THREAD

**Discussion by 120-cell** [ppy/osu#21571] [Ideas] [answered]
**Description:** An option to make sliders snake in at the same speed that they need to be followed at, like in this video: https://www.youtube.com/watch?v=sDmhTU9ytrk. They would appear similar to how streams currently do.

**Benefits:** Doing this would allow for certain tech maps with accelerating sliders to be sightread. It would help avoid breaking on sudden slow sliders, which can be very frustrating. It also just looks very nice in my opinion.

This could be implemented as a customizable speed for snaking sliders where the lowest possible value results in this behaviour, as a binary settings, or as a new mod.

**Comment by ghost** [comment, marked as answer]
duplicate of #13729
